proto/evmos/inflation/v1/genesis.proto

```diff
@@ -2,19 +2,41 @@ syntax = "proto3";
 package evmos.inflation.v1;
 
 import "gogoproto/gogo.proto";
+
 import "evmos/inflation/v1/inflation.proto";
 
 option go_package = "github.com/tharsis/evmos/x/inflation/types";
 
-// GenesisState defines the mint module's genesis state.
+// GenesisState defines the inflation module's genesis state.
 message GenesisState {
-  // minter is a space for holding current rewards information.
-  Minter minter = 1 [ (gogoproto.nullable) = false ];
-
   // params defines all the paramaters of the module.
-  Params params = 2 [ (gogoproto.nullable) = false ];
+  Params params = 1 [ (gogoproto.nullable) = false ];
+  // current period
+  uint64 period = 2;
```

Comment:
what's the period in this case? time? epoch?

Reply:
@fedekunze a period is by the `epochs_per_period` param. In our case we start with:

* `epoch_identifier: "day"
* `epochs_per_period`: 365
* `period`: 0

This means that a period ends after 365 daily epochs ( a year). We still, however, need to find a good solution to detect the correct period when we want to customize the params above.

Say we only change the `epochs_per_period` from 365 to 10 and the rest stays the same (`period` =  1, `epochNumber`= 400)  and  we check in the hooks if a period has passed with:
```
epochNumber > epochs_per_period * period
```
then it's likely that